src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1423,6 +1424,16 @@ private function getErrorMessages(Exception $e)
                 $e->getMessage(),
                 'Admin.Orderscustomers.Notification'
             ),
+            NegativeCartRuleValueException::class => [
+                NegativeCartRuleValueException::NEGATIVE_AMOUNT => $this->trans(
+                    'Reduction amount cannot be lower than zero.',
+                    'Admin.Orderscustomers.Notification'
+                ),
+                NegativeCartRuleValueException::NEGATIVE_PERCENTAGE => $this->trans(
```

Comment:
It's not only for negative value since you also check that it is inferior to 100 So you should rename this const to `NegativeCartRuleValueException::INVALID_PERCENTAGE` And for consistency the exception name should also be renamed to `InvalidCartRuleValueException`